.github/workflows/build.yml

```diff
@@ -12,6 +12,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
```

Comment:
Might be worth to port this patch on this file too since it also needs the version : https://github.com/Sygmei/ObEngine/blob/master/.github/workflows/artifacts.yml